blocks/elastic-content-strip/elastic-content-strip.js

```diff
@@ -0,0 +1,28 @@
+import { a } from '../../scripts/dom-helpers.js';
+
+import {
+  decorateInnerHrefButtonsWithArrowIcon,
+  decorateBlockMode,
+} from '../../scripts/scripts.js';
+
+export default function decorate(block) {
+  block.querySelectorAll('.elastic-content-strip > div > div').forEach((div) => {
+    const linkHref = div.querySelectorAll('a')[0].href;
+
+    const elasticContentWrapper = a({
+      class: 'elastic-content-link-wrapper',
+      href: linkHref,
+    });
+    div.parentNode.insertBefore(elasticContentWrapper, div);
+    elasticContentWrapper.appendChild(div);
+
+    decorateInnerHrefButtonsWithArrowIcon(elasticContentWrapper);
+  });
+
+  block.querySelectorAll('.button-container').forEach((bc) => {
+    bc.children[0].classList.remove('button');
+    bc.children[0].classList.add('learn-more-icon-container');
```

Comment:
Is this class meaningful? Why not just select .button-container > a in the css?

Reply:
Its two things, needs to remove generic button class and its nested 6 levels deep. 

<img width="1165" alt="image" src="https://github.com/user-attachments/assets/331a33d5-923f-4f21-a73e-4dd9ccc72f2d">

While the class is flat. We can ofc select everything, even if it is N-level deep, but it might not be worth the tradeoff of losing visibility and adding unnecessary nesting.